modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1322,61 +1324,61 @@ return view.extend({
 				crypto_modes.sort(function(a, b) { return b[2] - a[2] });
 
 				for (var i = 0; i < crypto_modes.length; i++) {
-					var security_level = (crypto_modes[i][2] >= 30) ? _('strong security')
-						: (crypto_modes[i][2] >= 20) ? _('medium security')
-							: (crypto_modes[i][2] >= 10) ? _('weak security') : _('open network');
+					var security_level = (crypto_modes[i][2] >= 30) ? _('High')
```

Comment:
No.
These explanations were added a while ago.  High/Medium/Low is much more vague the the current explanations.

Reply:
This change was to stop values being too large for the dropdown menu with after being selected and being truncated.

Reply:
Switched back to previous design which may truncate after being selected.